hashintel/hash
# Jump to start of document when pressing Enter in page's title

## 🌟 What is the purpose of this PR?



This PR makes it so the start of a document is focused when pressing the Enter key inside the Title of a page.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203179076056209/1203623179643222/f) _(internal)_


## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:












### 📜 Does this require a change to the docs?




The changes in this PR:






### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:






## ⚠️ Known issues


[Paragraph immediately loses focus in Chrome](https://app.asana.com/0/1203179076056209/1204610234070526/f)

## ❓ How to test this?



1.  Navigate to a page;
2.  Focus the title and press Enter;
3. Check that the first block in the document is focused.
